src/bn256/assembly.rs

```diff
@@ -1,5 +1,320 @@
 macro_rules! assembly_field {
-    ($field:ident, $modulus:ident, $inv:ident) => {
+    (
+        $field:ident,
+        $modulus:ident,
+        $inv:ident,
+        $modulus_str:ident,
+        $two_inv:ident,
+        $root_of_unity_inv:ident,
+        $delta:ident,
+        $zeta:ident
```

Comment:
I would suggest passing `R, R1, R2` as macro arguments to improve clarity.

Reply:
Fixed in https://github.com/privacy-scaling-explorations/halo2curves/pull/4/commits/5620c4c75dd444c2a7a5c24ca9eef7e82ecb2c4a.